reference-project/src/main/resources/application.properties

```diff
@@ -44,3 +44,8 @@ quarkus.micrometer.binder.http-server.enabled=true
 
 # TEST
 %test.quarkus.tkit.log.console.json=false
+
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.jdbc.url=jdbc:postgresql://localhost:5432/demo
+quarkus.datasource.username=demo
+quarkus.datasource.password=demo
```

Comment:
we need to remove this password stuff as this will ring some alarms.
Could we have datasource without password at all?

Reply:
Only the password or also the username? You can create a Postgres database without password and connect to it. I have tested this (username is required).
I think we could also switch to h2. In that case it would be easier to set up the whole application (no docker container with postgres db needed) and I think then we would only need to configure the url.

Reply:
H2 is one way. Alternatively, if we wish to keep Docker/postgres, we could use env vars or split application.properties into two files, where passwords, etc. are stored in another file (say pass.properties), which is included in application.properties and ignored in .gitignore

Reply:
I moved the credentials to a credentials provider class so that the user can configure the credentials himself by setting them as environment variables. If we change the datasource to h2 we also have to change the deployment files. So its easier this way.